src/components/error/Error.tsx

```diff
@@ -0,0 +1,9 @@
+import { Children } from "react";
+
+type Props = {
+  Children: string | undefined;
```

Comment:
- jeżeli chcesz coś ustawić jako undefined, nie musisz tego w ten sposób deifniować, wystarczy dodać przed dwukropkiem znak zapytania
- lepiej użyć wbudowanego propsa "children", otypować go jako ReactNode. Za jego pomocą złapiesz wszystko to, co będzie opakowane tagiem Twojego komponentu <Error> costam costam </Error>. Wtedy nie musisz przekazywać tego jako prop, tylko wkladasz to miedzy znaczniki.